values.yaml

```diff
@@ -291,6 +291,8 @@ elasticsearch:
   port: 9200
   version: "v7"
   logLevel: "error"
+  authentication:
+    enabled: true
```

Comment:
flip option to `.authentication.disabled` instead - should be enabled by default and if we want to disable this set to true